syntax/log.vim

```diff
@@ -92,13 +92,13 @@ syn keyword logLvFatal      FATAL
 syn keyword logLvEmergency  EMERGENCY EMERG
 syn keyword logLvAlert      ALERT
 syn keyword logLvCritical   CRITICAL CRIT
-syn keyword logLvError      ERROR ERRORS ERR E
+syn keyword logLvError      ERROR ERRORS ERR E error
 syn keyword logLvFailure    FAILURE FAILED FAIL F
-syn keyword logLvWarning    WARNING WARN W
+syn keyword logLvWarning    WARNING WARN W warning warn
 syn keyword logLvNotice     NOTICE
-syn keyword logLvInfo       INFO I
-syn keyword logLvDebug      DEBUG DBG D
-syn keyword logLvTrace      TRACE
+syn keyword logLvInfo       INFO I info
+syn keyword logLvDebug      DEBUG DBG D debug dbg
+syn keyword logLvTrace      TRACE trace
```

Comment:
I'm fine with most of the changes here, but I'm personally against to `trace`  because:  
(1) the lowercase `trace` is relatively common to see in particular types of log, at least the ones that I work with, e.g. `call trace`, `stack trace`, `trace event` etc. (see `samples/`)  
(2) unlike others (`info` and above) that highlight a word (which is not that bad to see in sentences, sometimes even makes more sense), applying syntax to lowercase `trace` usually dims or grays out the word in color themes, and it's annoying to look at a line with a dimmed word as I need to focus and "look into" the visual pit to pick up the piece of a sentence. I know it's mostly a matter of personal preference, but at least that's how I feel after applying this locally.

So... I'd be happy to take this if `trace` is excluded. What do you think?

Reply:
I kind of agree, provided the 'trace' level is usually the most common one and shouldn't bring any special attention. I'll remove it

Reply:
Thanks. Could you mark this as ready to review so that I can merge?

Reply:
Never mind, I merged the PR.